content/en/network_monitoring/devices/snmp_traps.md

```diff
@@ -160,12 +159,13 @@ You can write these mappings by hand, or generate mappings from a list of MIBs u
 - [`ddev`][4] (`pip3 install "datadog-checks-dev[cli]"`)
 - [`pysmi`][5] (`pip3 install pysmi`)
 
-Put all your MIBs into a dedicated folder. Then, run:
-`ddev meta snmp generate-traps-db -o ./output_dir/ /path/to/my/mib1 /path/to/my/mib2 /path/to/my/mib3 ...`
+Place all your MIBs in a dedicated folder and run:
 
-If your MIBs have dependencies, `ddev` fetches them online if they can be found.
+```
```

Comment:
```suggestion
```shell
```

this'll format the command so that it looks like a shell command (prepended with `$`) and also add a Copy button on mouseover (the `$` doesn't get copied)